<launch>

  <arg name="stateEstimationTopic" default="/state_estimation"/>
  <arg name="trackingCamBackward" default="false"/>
  <arg name="trackingCamXOffset" default="0.0"/>
  <arg name="trackingCamYOffset" default="0.0"/>
  <arg name="trackingCamZOffset" default="0.0"/>
  <arg name="trackingCamScale" default="1.0"/>
  <arg name="shiftGoalAtStart" default="false"/>
  <arg name="goalYaw" default="0.0"/>
  <arg name="goalX" default="0.0"/>
  <arg name="goalY" default="0.0"/>
  <arg name="goalZ" default="1.0"/>

  <node pkg="control_tuner" exec="controlTuner" name="controlTuner" output="screen">
    <param name="stateEstimationTopic" value="$(var stateEstimationTopic)" />
    <param name="pubSkipNum" value="1" />
    <param name="trackingCamBackward" value="$(var trackingCamBackward)" />
    <param name="trackingCamXOffset" value="$(var trackingCamXOffset)" />
    <param name="trackingCamYOffset" value="$(var trackingCamYOffset)" />
    <param name="trackingCamZOffset" value="$(var trackingCamZOffset)" />
    <param name="trackingCamScale" value="$(var trackingCamScale)" />
    <param name="lookAheadScale" value="0.2" />
    <param name="minSpeed" value="0.5" />
    <param name="maxSpeed" value="2.0" />
    <param name="velXYGain" value="0.3" />
    <param name="posXYGain" value="0.05" />
    <param name="stopVelXYGain" value="0.2" />
    <param name="stopPosXYGain" value="0.2" />
    <param name="smoothIncrSpeed" value="0.75" />
    <param name="maxRollPitch" value="30.0" />
    <param name="yawGain" value="2.0" />
    <param name="maxRateByYaw" value="60.0" />
    <param name="posZGain" value="1.5" />
    <param name="maxVelByPosZ" value="0.5" />
    <param name="manualSpeedXY" value="2.0" />
    <param name="manualSpeedZ" value="1.0" />
    <param name="manualYawRate" value="60.0" />
    <param name="stopDis" value="0.5" />
    <param name="slowDis" value="2.0" />
    <param name="joyDeadband" value="0.1" />
    <param name="fixGoalYaw" value="true" />
    <param name="shiftGoalAtStart" value="$(var shiftGoalAtStart)" />
    <param name="goalYaw" value="$(var goalYaw)" />
    <param name="goalX" value="$(var goalX)" />
    <param name="goalY" value="$(var goalY)" />
    <param name="goalZ" value="$(var goalZ)" />
  </node>

</launch>
